python/docs/introduction.rst

```diff
@@ -47,24 +47,34 @@ Key Concepts
 
 
 
-* :class:`.SQLContext`
-   - This represents the current context of your SQL program, data sources
-     and sinks. In Feldera terminology, this represents both a Program and a
-     Pipeline.
+* :class:`.PipelineBuilder`
+    - This is the main class that you will use to build your Feldera pipeline.
+    - It is a builder for creating a Feldera pipeline.
+    - It provides 3 methods for building a pipeline:
 
-   - Example:
+        - :meth:`.PipelineBuilder.create`
+        - :meth:`.PipelineBuilder.create_or_replace`
+        - :meth:`.PipelineBuilder.get`
```

Comment:
I don't think `get` should be a method of the builder, since we're not creating anything, but simply connect to an existing pipeline.

Reply:
I was stuck in two minds here. 

`PipelineBuilder.get` or `Pipeline.get(name, client)`

I put it in `PipelineBuilder` for the sake of consistency. 
Should I move it to `Pipeline`?